app/utils/misc.ts

```diff
@@ -0,0 +1,51 @@
+export function typedBoolean<T>(
+  value: T,
+): value is Exclude<T, "" | 0 | false | null | undefined> {
+  return Boolean(value);
+}
+
+export function getDomainUrl(request: Request) {
+  const host =
+    request.headers.get("X-Forwarded-Host") ?? request.headers.get("host");
+  if (!host) {
+    throw new Error("Could not determine domain URL.");
+  }
+  const protocol = host.includes("localhost") ? "http" : "https";
+  return `${protocol}://${host}`;
+}
+
+export function removeTrailingSlash(s: string) {
+  return s.endsWith("/") ? s.slice(0, -1) : s;
+}
+
+/**
+ * Simple deep equality checker. It will check value of every field in each object, as well as the length of their key sets.
+ */
+export function isEqual<T>(entry1: T, entry2: T): boolean {
```

Comment:
Siden den krever to objekter, kan det være greit å gjøre det tydligere
```suggestion
export function isDeepEqual<T>(entry1: T, entry2: T): boolean {
```

💭 
Går det an å enforce i type-deklarasjonen? 🤔 Ikke bruk tid på for min del, bare en tanke.

Reply:
💭 
Eller kanskje det ikke er noe vits? Eneste grunnen man skal bruke denne funksjonen er for å sammenligne objekter. Er det ikke objekter så bruker man jo `===`.

Med mindre man har en generisk funksjon som kan få inn både objekter og primitiver 🤔

Reply:
Så at jeg bare måtte flytte litt om på koden for å støtte primitive argumenter også, så tenkte det var greiest, om vi har ting vi ikke er sikre på hva er 👌 [1c81fb7](https://github.com/capraconsulting/nettsiden/pull/6/commits/1c81fb7a55513accc52eb0fdf0235455af9bc9f8)

Reply:
Ah nice 🥳